### PR TITLE
#167727749 Travel admin can create accommodation

### PR DESCRIPTION
**Pull request by tobslob**
## What does this PR do?
- This PR enable to travel admin to create an accommodation

## Description of tasks to be done

- create accommodation controller
- create accommodation route
- create accommodation models
- implement cloudinary and multer for accommodation images
- validate controller inputs
- write unit and integration test

## How to manually test

- Clone the repo
- Run ``` npm install ```
- run ``` npm run seed ```
- Using postman or any http request tool, make a post request to /api/v1/requests/accommodation to create accommodation
- Navigate to /api/v1/docs to test further with swagger ui

<a href="https://www.pivotaltracker.com/story/show/167727749"> [Finishes #167727749] </a>

### CODE REVIEW

File: src/models/accommodation.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+module.exports = (sequelize, DataTypes) => {
```
Comment (tobslob) [inline src/models/accommodation.js:1]: 1. remove createdAt and UpdatedAt
2. Update swagger endpoint to take in image

File: src/controllers/accommodationController.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+const createAccommodation = async (req, res) => {
+  let images;
+  try {
+    const { body } = req;
+    const {
+      country, city, address, accommodation, accommodationType, roomType, numOfRooms,
+      description, facilities
+    } = body;
+
+    if (req.files) {
```
Comment (tobslob) [inline src/controllers/accommodationController.js]: 1. set allowNull to false in table

**Review by komsic — APPROVED** [review]
LGTM

File: src/models/accommodation.js (hunk 1 above)
Comment (tobslob) [reply src/models/accommodation.js:1]: Done

**Review by ladaposamuel — APPROVED** [review]
Looks good to me

**Pull request merged**